server/broadcast.go

```diff
@@ -216,6 +216,16 @@ func selectSession(sessions []*BroadcastSession, exclude []*BroadcastSession, du
 		// threshold is selectable
 		if len(session.SegsInFlight) == 0 {
 			if session.LatencyScore > 0 && session.LatencyScore <= SELECTOR_LATENCY_SCORE_THRESHOLD {
+				clog.PublicInfof(ctx,
+					"Selected orchestrator reason=%v, eth-address=0x%v, ip-address=%v",
```

Comment:
It might be better to put `reason=` at the end of log as it contains commas and spaces. It might be easier for reading and parsing.

Reply:
Even better, can we have these as key/value fields rather than baked into the log line?

Reply:
@thomshutt you mean to extract `eth-address` and `ip-address` as promtail/loki labels? We could log them as string in broadcaster and then add regex logic in `promtail` to extract `eth-address` and `ip-address` as labels.

Reply:
For what its worth, I updated to same approach.  I did some conformity updates it commits after this one that better followed the clog presentation.  Also added some updates to return information to level where clog context was active (e.g. selection) so could have better conformity.

https://github.com/ad-astra-video/go-livepeer/commit/af65b34728eec99189c4e301c8f7be6f2469253f

https://github.com/ad-astra-video/go-livepeer/commit/9d8d89b10a6ba81b4761d3bd32192d5bce7a7353

Reply:
@ad-astra-video: for this initial PR, I opted for simpler logs. Several unit tests were failing due to code races when attempting to log certain values (e.g. info related to payment tickets) where it was causing certain http responses to time out. 

Also unit tests caught a few panics which were getting triggered when trying to log values like `ethcommon.Bytes2Hex(session.OrchestratorInfo.TicketParams.Recipient)` when no segments were in-flight -- this was being caused by a null `session.OrchestratorInfo` object under certain conditions. I'm not fully ramped up on this code base yet so didn't quite understand all the conditions that can lead to this. I'll be spending some more time on this to get a better understanding. 

Once we verify logging to the public loki instance is working, we can follow up with further changes that will log all the values that will be most beneficial.